packages/react/src/components/Storage/FileUploader/hooks/useFileUploader/types.ts

```diff
@@ -17,3 +17,14 @@ export interface UseFileUploader extends DragActionHandlers {
   setInDropZone?: React.Dispatch<React.SetStateAction<boolean>>;
   addTargetFiles?: (targetFiles: File[]) => number;
 }
+
+export interface UseFileUploaderProps {
+  maxFileSize: number;
+  acceptedFileTypes: string[];
+  allowMultipleFiles: boolean;
+  isLoading: boolean;
+}
+
+export interface UseFileUploader {
```

Comment:
Nit: we use hook PascalCase for the return type for a fair amount of our existing hooks. Using `UseFileUploader` to type the function breaks convention

Reply:
good catch, will update